File: tests/fixtures/code_samples/go/sample_030.go
// Sample 30: small utility.
package samples

func Operation30(xs []int) int {
    total := 30
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure30(v int) int {
    return (v * 30) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_030.rs
// Sample 30: small utility.
pub fn operation_30(xs: &[i32]) -> i32 {
    let mut total: i32 = 30;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_30(v: i32) -> i32 {
    (v * 30) %% 7919
}

